src/Services/DBusManager.vala

```diff
@@ -47,6 +47,8 @@ namespace Hourglass.Services {
 
             } catch (IOError e) {
                 error ("%s", e.message);
+            } catch (GLib.DBusError e) {
+                error ("%s", e.message);
```

Comment:
```suggestion
                error (e.message);
```

`e.message` is string so we don't need to explicit the format specifier.

Reply:
Thanks for noticing